app/screens/emoji_picker/index.tsx

```diff
@@ -14,37 +14,29 @@ type Props = {
     closeButton: never;
 };
 
+const EMOJI_PICKER_BUTTON = 'close-add-reaction';
+
 const EmojiPickerScreen = ({closeButton, componentId, onEmojiPress}: Props) => {
     useEffect(() => {
         setButtons(componentId, {
             leftButtons: [
                 {
                     icon: closeButton,
-                    id: 'close-add-reaction',
+                    id: EMOJI_PICKER_BUTTON,
                     testID: 'close.emoji_picker.button',
                 },
             ],
             rightButtons: [],
         });
-
-        const unsubscribe = Navigation.events().registerComponentListener({
-            navigationButtonPressed: ({buttonId}: { buttonId: string }) => {
-                if (buttonId === 'close-add-reaction') {
-                    close();
-                }
-            },
-        }, componentId);
-
-        return () => {
-            unsubscribe.remove();
-        };
     }, []);
 
     const close = useCallback(() => {
```

Comment:
I know is not part of the PR, but looking at this function I don't think it needs memoization, also dismissModal should include the componentId to be more certain

Reply:
done :)